Sources/TensorFlow/Operators/NN.swift

```diff
@@ -138,6 +138,132 @@ public extension Padding {
     }
 }
 
+func _vjpConv2D<Scalar: TensorFlowFloatingPoint>(
```

Comment:
All VJP functions should have a `@usableFromInline` attribute. Same elsewhere.
```suggestion
@usableFromInline
func _vjpConv2D<Scalar: TensorFlowFloatingPoint>(
```